app/assets/javascripts/application.js

```diff
@@ -136,6 +134,15 @@ function onContentLoad(){
   })
 }
 
+function activateDatatables() {
+  $('[data-table=inline]').not('.dataTable').dataTable(
+      {
+        "sDom": "<'row'<'col-md-6'f>r>t<'row'<'col-md-6'i><'col-md-6'p>>",
+        "sPaginationType": "bootstrap"
+      }
+  );
+}
```

Comment:
this is currently used both by my PR and in the compute resource view. Would it be possible for you to use a different selector then `[data-table=inline]`?